assets/src/edit-story/elements/text/edit.js

```diff
@@ -253,15 +253,16 @@ function TextEdit({
     editorRef.current.focus();
   }, []);
 
+  const { fontFamily } = rest;
+
   // Remeasure the height on each content update.
   useEffect(() => {
     const wrapper = wrapperRef.current;
     const textBox = textBoxRef.current;
     editorHeightRef.current = textBox.offsetHeight;
     wrapper.style.height = `${editorHeightRef.current}px`;
-  }, [editorState, elementHeight]);
+  }, [editorState, elementHeight, fontFamily]);
```

Comment:
As mentioned before, more than just fontFamily are needed here. Not sure if there's a clean way to generalize this. E.g. we want to catch any height-affecting properties.